spec/acceptance/hieradata/common.yaml

```diff
@@ -0,0 +1,12 @@
+---
+foreman::client_ssl_ca: /etc/foreman/proxy_ca.pem
+foreman::client_ssl_cert: /etc/foreman/client_cert.pem
+foreman::client_ssl_key: /etc/foreman/client_key.pem
+foreman::initial_location: Default Location
+foreman::initial_organization: Default Organization
```

Comment:
Do we really care about these? Not setting these results in the Foreman built in defaults being used, which are `Default location` and `Default organization`:
https://github.com/theforeman/foreman/blob/3ff1fd27fa32f7a52ecd995417521b5b91c9a093/db/seeds.d/050-taxonomies.rb#L9